ui/src/app/components/source-details-form/source-details-form.component.html

```diff
@@ -10,15 +10,23 @@
     <div class="form-container">
         <div *ngIf="(selectedOption === 'dumpFile')">
             <form [formGroup]="dumpFileForm" (ngSubmit)="setSourceDBDetailsForDump()">
-                <h4>Load from database dump</h4>
+                <h4 class="primary-header">Load from database dump</h4>
                 Dump File
-                <mat-form-field class="full-width" appearance="outline">
+                <mat-form-field appearance="outline">
                     <mat-label>File path</mat-label>
-                    <input matInput placeholder="File path" name="filePath" type="text" formControlName="filePath" />
+                    <input matInput placeholder="File path" name="filePath" type="text" formControlName="filePath"
+                        readonly (click)="file.click()" />
+
+                    <mat-spinner matSuffix *ngIf="uploadStart && !uploadSuccess && !uploadFail" [diameter]="25">
```

Comment:
Curious: The static `diameter` value of `25`, does it scale automatically when viewing in screens of different sizes?

Reply:
This doesn't scale based on screen size